### PR TITLE
Finishing touch Frank!Doc issue on DocumentedEnum

### PR DESCRIPTION
**Pull request by mhdirkse**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java [hunk 1]
```diff
@@ -119,7 +119,7 @@
 	private int correlationIdMaxLength = -1;
 
 	public enum AcknowledgeMode implements DocumentedEnum {
-		@EnumLabel("") NOT_SET(0),
+		@EnumLabel("none") NOT_SET(0),
 		@EnumLabel("auto") AUTO_ACKNOWLEDGE(Session.AUTO_ACKNOWLEDGE),
 		@EnumLabel("client") CLIENT_ACKNOWLEDGE(Session.CLIENT_ACKNOWLEDGE),
 		@EnumLabel("dups") DUPS_OK_ACKNOWLEDGE(Session.DUPS_OK_ACKNOWLEDGE);
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java:125]: Voor de volledigheid zou hier nog javadoc bij kunnen/moeten met de beschrijving die je op regel 828 weg hebt gehaald. Ook is het handig als er een meer geschrijvende tekst bij komt zoals:

`/** auto or auto_acknowledge: Specifies that the session is to automatically acknowledge consumer receipt of messages when message processing is complete. */ `

`/** dups or dups_ok_acknowledge: Specifies that the session is to "lazily" acknowledge the delivery of messages to the consumer. "Lazy" means that the consumer can delay the acknowledgment of messages to the server until a convenient time; meanwhile the server might redeliver messages. This mode reduces the session overhead. If JMS fails, the consumer may receive duplicate messages. */ `

`/** client or client_acknowledge: Specifies that the consumer is to acknowledge all messages delivered in this session. */ `

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java (hunk 1 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java:125]: Er moet zowel javaDoc bij de class, die dan beschrijft wat AcknowledgeMode is, en bij elke waarde, die die specifieke waarde beschrijft

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

File: core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java [hunk 2]
```diff
@@ -820,12 +820,12 @@ public void setAckMode(int ackMode) {
...
 	}
 
-	public AcknowledgeMode getAckModeEnum() {
+	public AcknowledgeMode getAcknowledgeModeEnum() {
 		return ackMode;
 	}
 
 
-	@IbisDoc({"3", "Acknowledge mode, can be one of ('auto' or 'auto_acknowledge'), ('dups' or 'dups_ok_acknowledge') or ('client' or 'client_acknowledge')", "auto_acknowledge",})
+	@IbisDoc({"3", "Acknowledge mode", "auto"})
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java:828]: `If not transacted, the way the application informs the JMS provider that it has successfully received a message.`

Comment (mhdirkse) [reply core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java]: Gedaan.

**Review by nielsm5 — APPROVED** [review]

**Comment by mhdirkse** [comment]
Please merge https://github.com/ibissource/iaf/pull/2136 first. After this merge, I will merge the master to this pull request manually.

**Review by gvanbrakel — APPROVED** [review]

**Pull request merged**